docs/Developers/Newcomers_Guide.rst

```diff
@@ -293,17 +304,18 @@ Now there's two possibilities:
 .. note::
 
     Wait until the reviewer has already reviewed your whole Pull Request
-    and has labeled it ``process/wip``. Else, if you push again and his
-    comments disappear, it can be considered rude.
+    and has labeled it ``process/wip``.
+    Else, if you push again and his comments disappear, it can be considered
+    rude.
 
 .. note::
 
     You might be wondering what those CI things on your ``Pull Request`` are.
     For more detailed info about them, see `this page`_.
 
 It's highly unlikely that your ``Pull Request`` will be accepted on the first
-attempt - but don't worry, that's just how it works. It helps us maintain
-coala **clean** and **stable**.
+attempt - but don't worry, that's just how it works.
+It helps us maintain coala **clean** and **stable**.
 
 .. seealso::
```

Comment:
this link in this seealso should be mentioned earlier at the place where we tell people to review themselves